main/src/ca/uwaterloo/flix/language/phase/Lexer.scala

```diff
@@ -352,7 +362,13 @@ object Lexer {
       case _ if isOperator(":") => TokenKind.Colon
       case _ if isOperator("**") => TokenKind.StarStar
       case _ if isOperator("<-") => TokenKind.ArrowThinL
-      case _ if isOperator("->") => TokenKind.ArrowThinR
+      case _ if isOperator("->") =>
+        // Check for whitespace around arrow.
```

Comment:
I think we want to explain here. A `->` string is a function arrow if there is whitespace before and after? Otherwise it is a structarrow.

So `foo ->a` is struct arrow?

Actually, in the comment, could you just write out all the cases with examples. Thanks!

Reply:
Wait, it says or, so maybe I am confused. Definitely want a small table and better description 👍